scripts/st2bootstrap-deb.sh

```diff
@@ -1,145 +1,165 @@
 #!/bin/bash
 
+set -e
+
 fail() {
-	echo "############### ERROR ###############"
-	echo "# Failed on $1 #"
-	echo "#####################################"
-	exit 2
+  echo "############### ERROR ###############"
+  echo "# Failed on $1 #"
+  echo "#####################################"
+  exit 2
+}
+
+install_st2_dependencies() {
+  sudo apt-get update
+  sudo apt-get install -y curl mongodb-server rabbitmq-server
 }
 
-install_dependencies() {
-	sudo apt-get update
-	sudo apt-get install -y mongodb-server rabbitmq-server postgresql
+install_st2() {
+  # Following script adds a repo file, registers gpg key and runs apt-get update
+  curl -s https://packagecloud.io/install/repositories/StackStorm/staging-stable/script.deb.sh | sudo bash
+  sudo apt-get install -y st2
+  sudo st2ctl reload
+  sudo st2ctl start
 }
 
-setup_repositories() {
-    # Following script adds a repo file, registers gpg key and runs apt-get update
-    curl -s https://packagecloud.io/install/repositories/StackStorm/staging-stable/script.deb.sh | sudo bash
+configure_st2_user () {
+
+  # Create an SSH system user
+  sudo useradd stanley
```

Comment:
You need to verify if user is already created (and it's created after `st2` package installed):
https://github.com/StackStorm/st2-packages/blob/master/scripts/st2bootstrap-el7.sh#L36-L38